### PR TITLE
Added parameter to ignore NPM

### PR DESCRIPTION
**Pull request by Mte90**
Ref: https://github.com/Varying-Vagrant-Vagrants/VVV/issues/2584

### CODE REVIEW

File: provision/vvv-init.sh [hunk 1]
```diff
@@ -177,8 +178,10 @@ else
   handle_git_wp
 fi
 
-try_npm_install
-try_grunt_build
+if [[ "${NPM}" == "true" ]]; then
+  try_npm_install
+  try_grunt_build
+fi
```
Comment (tomjn) [inline provision/vvv-init.sh:184]: an else statement reporting that it was skipped would be good here

Comment (tomjn) [inline provision/vvv-init.sh:184]: Perhaps separate flags for grunt and npm?

**Review by tomjn — CHANGES_REQUESTED** [review]
I wonder if we should also do some UX changes, in particular making `npm` failure non-blocking, so if `npm` fails then we can catch and report this to the user and mention the new flags so it isn't a mystery?

Maybe not this but something similar:

> `npm` failed to run, check the site log for output in the provisioner logs folder. You may want to run `npm` commands on the host and disable npm during provisioning. We'll continue provisioning but some things may not work.

File: provision/vvv-init.sh (hunk 1 above)
Comment (Mte90) [reply provision/vvv-init.sh:184]: Grunt packages are in the same package.json file so it isn't possible to install just them.

**Comment by Mte90** [comment]
> I wonder if we should also do some UX changes, in particular making `npm` failure non-blocking, so if `npm` fails then we can catch and report this to the user and mention the new flags so it isn't a mystery?

I don't know if it is easy to track NPM errors or if return a specific exit code

**Comment by Mte90** [comment]
I see also another issue in the tests config file that is not filled with the VVV data.
I will update this PR.

**Comment by tomjn** [comment]
True but I was referring to running the grunt build

**Comment by tomjn** [comment]
Just did the npm commands return a good or a bad code, we don’t need to be
any more specific than that

**Comment by Mte90** [comment]
You are right, I checked now the code and I saw that grunt was executed twice in the provision.
The last commit should fix everything, I am testing it.

**Comment by Mte90** [comment]
@tomjn can you check now this pr?

**Review by tomjn — APPROVED** [review]

**Pull request merged**